### ISSUE TITLE
Archive Document on Merge

### ISSUE THREAD

**Issue by DanielHabenicht** [luke-browning/pdf-web-edit#54]
After Merging a document it would be nice if the document that got merged would be moved to the archive automatically. 
![image](https://github.com/luke-browning/pdf-web-edit/assets/13590797/df07e097-8d33-4ccf-9347-55344580e650)

Currently, it is retained in the inbox and has to be removed manually. 

Should it be configurable in the options?

**Comment by luke-browning** [comment]
Good idea! And yes, the behaviour should be configurable in the options.

**Comment by Masgalor** [comment]
I am a little confused by the fact, that it is possible to merge a document into itself.
If I hit the merge button the following selection of documents always includes the target document.

So if I merged a document into itself I simply got every page in that document duplicated.
With this change in place I get an error message and my document is moved from my inbox to the archive.
I think we need a check in place to avoid merging a document into itself before this can be enabled.
